### PR TITLE
Doc changes for v0.5.0

### PR DESCRIPTION
**Pull request by StrongMonkey**
Issue: https://github.com/rancher/rio/issues/512

### CODE REVIEW

**Comment by StrongMonkey** [comment]
@daxmc99 Please review, `rio uninstall` has been added.

File: docs/README.md [hunk 1]
```diff
@@ -26,34 +26,31 @@
...
 * `https-port`: HTTPS port gateway service will listen. You can only set the HTTPS port if install mode is svclb or hostport. Default HTTPS port for svclb and hostport mode is 9443. If install mode is ingress, HTTPS port is determined by ingress and cannot be changed by Rio installer. Ingress controllers typically expose HTTPS port 443, although some ingress controllers allow you to specify custom HTTPS ports.
 * `ip-address`: Rio generates DNS domains that map to IP address of the gateway. Rio will attempt to detect gateway IP addresses automatically, and you can override by manually specify comma-separated IP addresses.
-* `service-cidr`: Rio will attempt to detect service CIDR for service mesh intercept traffic. You can override by manually specify a service CIDR.
 * `disable-features`: Specify feature to disable during install. Here are the available feature list.
 
 | Feature | Description |
 |----------|----------------|
 | autoscaling | Auto-scaling services based on QPS and requests load
 | build | Rio Build, from source code to deployment
-| grafana | Grafana Dashboard
```
Comment (daxmc99) [inline docs/README.md]: we are dropping grafana here but we mention it later in the doc?

File: docs/README.md [hunk 2]
```diff
@@ -26,34 +26,31 @@
...
-| istio | Service routing using Istio
-| kiali | Kiali Dashboard
+| gateway | Gateway service based on pilot and envoy
+| istio | Istio service mesh
+| linkerd | Linkerd service mesh
 | letsencrypt | Let's Encrypt
-| mixer | Istio Mixer telemetry
-| prometheus | Enable prometheus
```
Comment (daxmc99) [inline docs/README.md]: Same as above

File: docs/README.md [hunk 3]
```diff
@@ -26,34 +26,31 @@
...
 | rdns | Assign cluster a hostname from public Rancher DNS service
 
 * `httpproxy`: Specify HTTP_PROXY environment variable for Rio control plane. This is useful when Rio is installed behind an HTTP firewall.
-* `lite`: Disable all monitoring features including prometheus, mixer, grafana and kiali.
+* `lite`: Disable all monitoring features including prometheus, mixer, grafana and kiali. Only works in istio mesh mode.
```
Comment (cbron) [inline docs/README.md]: More of a question, linkerd also installs prometheus, so this flag with linkerd mode wouldn't disable the linkerd monitoring ?

File: docs/README.md [hunk 4]
```diff
@@ -26,34 +26,31 @@
...
 * `https-port`: HTTPS port gateway service will listen. You can only set the HTTPS port if install mode is svclb or hostport. Default HTTPS port for svclb and hostport mode is 9443. If install mode is ingress, HTTPS port is determined by ingress and cannot be changed by Rio installer. Ingress controllers typically expose HTTPS port 443, although some ingress controllers allow you to specify custom HTTPS ports.
 * `ip-address`: Rio generates DNS domains that map to IP address of the gateway. Rio will attempt to detect gateway IP addresses automatically, and you can override by manually specify comma-separated IP addresses.
-* `service-cidr`: Rio will attempt to detect service CIDR for service mesh intercept traffic. You can override by manually specify a service CIDR.
 * `disable-features`: Specify feature to disable during install. Here are the available feature list.
```
[the remark this answers is not in the document]
Comment (StrongMonkey) [reply docs/README.md]: We still need that to turn off other features like rdns and letsencrypts

File: docs/README.md (hunk 3 above)
Comment (StrongMonkey) [reply docs/README.md]: Yes, it won't disable linkerd monitoring. The original intention to use lite is that istio install is too large so we have to break it apart.

File: docs/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (cbron) [reply docs/README.md]: I meant for those specific features

[the remark this answers is not in the document]
Comment (cbron) [reply docs/README.md]: resolved offline

**Pull request merged**